client/app/components/QueryEditor.jsx

```diff
@@ -217,6 +217,12 @@ class QueryEditor extends React.Component {
     localOptions.set('liveAutocomplete', state);
   };
 
+  componentDidUpdate = () => {
+    // Work-around for a resizing issue, see https://github.com/getredash/redash/issues/3353
```

Comment:
```suggestion
    // ANGULAR_REMOVE_ME  Work-around for a resizing issue, see https://github.com/getredash/redash/issues/3353
```

(so we remember to get rid of this)